contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,95 @@
+---
+date: 2022-03-23
+title: Introducing Data Management in PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates", "Launch week"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the double-digit billions. In just the last year (since we started measuring total events), we've gone from ~8B to ~36.5B total events ingested in PostHog Cloud!
```

Comment:
```suggestion
PostHog is growing fast. In just the last year (since we started measuring total events), we've gone from ~8B to ~36.5B total events ingested in PostHog Cloud alone!
```